src/test/java/io/strimzi/test/container/AbstractIT.java

```diff
@@ -41,7 +41,9 @@ protected static Stream<Arguments> retrieveKafkaVersionsFile() throws IOExceptio
     }
 
     protected void assumeDocker() {
-        Assumptions.assumeTrue(System.getenv("DOCKER_CMD") == null || "docker".equals(System.getenv("DOCKER_CMD")));
+        Assumptions.assumeTrue(
+            System.getenv("DOCKER_CMD") == null ||
+                ("docker".equals(System.getenv("DOCKER_CMD")) || "podman".equals(System.getenv("DOCKER_CMD"))));
```

Comment:
What exactly is the purpose of this? Why do we check some environment variable like this?

Reply:
Well as I said I am not sure what was the first intention with this. I can remove this easily... :). Because I also don't see any strong benefit of such check...

Reply:
+1 for removing it

Reply:
Yeah ... if we do not know why we have it and it doesn't seem to make sense, let's remove it I guess.

Reply:
Removed.